src/helpers/paths.ts

```diff
@@ -0,0 +1,37 @@
+import * as ts from 'typescript';
+import * as path from 'path';
+
+import { PathInfo } from '../module-info';
+import { fixPath } from './fix-path';
+
+export function getEffectivePaths(compilerOptions: ts.CompilerOptions): PathInfo[] | undefined {
+	const baseUrl = compilerOptions.baseUrl;
+	const paths = compilerOptions.paths;
+
+	if (baseUrl === undefined || paths === undefined) {
+		return undefined;
+	}
+
+	const effectivePaths: PathInfo[] = [];
+
+	for (const moduleName in paths) {
+		if (paths[moduleName]) {
+			paths[moduleName].forEach((modulePath: string) => effectivePaths.push({
```

Comment:
```suggestion
			effectivePaths.push(...paths[moduleName].map((modulePath: string) => {
				moduleName,
				modulePath: fixPath(path.normalize(path.join(baseUrl, modulePath))),
			});
```

Reply:
```suggestion
			effectivePaths.push(...paths[moduleName].map((modulePath: string) => ({
				moduleName,
				modulePath: fixPath(path.normalize(path.join(baseUrl, modulePath))),
			})));
```

The brackets are still necessary to distinguish the object creation from a block.

Reply:
Oh, sure, you're right!